### ISSUE TITLE
Possible false positive in <script> multi-line comments containing leading asterisks, text, and slash

### ISSUE THREAD

**Issue by FiniteFractalSurface** [validator/validator#2122]
### Summary

A bit convoluted but an error is reported when a `<script>` element with a `src` attribute has as its content a multi-line comment with leading asterisks combined with slash (see example snippet). 

This seems like a false positive because I believe leading asterisks + text + slash in this context are allowed by [specs](https://html.spec.whatwg.org/multipage/scripting.html#inline-documentation-for-external-scripts), as they would be anywhere else (e.g. `<script>` without `src` atribute).

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<!doctype html>
<html lang="en">
 <head>
  <title>test</title>
  <script src="test.js">
   /*
   * This is a leading asterisk comment containing a forward slash / somewhere in the text
   */
  </script>
  <script src="test.js">
   /*
   * It of course also breaks on urls, see https://validator.w3.org/
   */
  </script>
 </head>
</html>
```

### Checker error messages

```shell
Error: The text content of element script was not in the required format: Expected space, tab, newline, or slash but found s instead.

Error: The text content of element script was not in the required format: Expected asterisk or slash but found v instead.
```

### Spec links/citations

https://html.spec.whatwg.org/multipage/scripting.html#inline-documentation-for-external-scripts

### Steps to reproduce

Validate code as shown in snippet above.

### Expected behavior

No errors. 

### Actual behavior

Errors as shown above.

**Comment by sideshowbarker** [comment]
Nice catch — thanks for taking time to report it~

https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFkb2N0eXBlIGh0bWw%2BCjxodG1sIGxhbmc9ImVuIj4KIDxoZWFkPgogIDx0aXRsZT50ZXN0PC90aXRsZT4KICA8c2NyaXB0IHNyYz0idGVzdC5qcyI%2BCiAgIC8qCiAgICogVGhpcyBpcyBhIGxlYWRpbmcgYXN0ZXJpc2sgY29tbWVudCBjb250YWluaW5nIGEgZm9yd2FyZCBzbGFzaCAvIHNvbWV3aGVyZSBpbiB0aGUgdGV4dAogICAqLwogIDwvc2NyaXB0PgogIDxzY3JpcHQgc3JjPSJ0ZXN0LmpzIj4KICAgLyoKICAgKiBJdCBvZiBjb3Vyc2UgYWxzbyBicmVha3Mgb24gdXJscywgc2VlIGh0dHBzOi8vdmFsaWRhdG9yLnczLm9yZy8KICAgKi8KICA8L3NjcmlwdD4KIDwvaGVhZD4KPC9odG1sPgo%3D
